src/components/AccountSelector.tsx

```diff
@@ -20,17 +20,19 @@ const Main = () => {
   const { activeAccount, setActiveAccount, accounts } = useAccount()
 
   const onChange = (account: HTMLInputElement) => {
-    setActiveAccount(account.innerText)
-    localStorage.setItem("activeAccount", account.innerText)
+    const activeAccount = accounts.filter(acc => acc.name && acc.name.includes(account.innerText))
```

Comment:
It seems strange that we're passing an `HTMLInputElement` here and getting its `innerText`.

Let us set the `eventKey` on the `Dropdown.Item` and get it on the `onSelect` event. Details here:

https://react-bootstrap.github.io/components/dropdowns/#dropdown-item-props

I am available for a pair tonight :D